### PR TITLE
fixed video source serialization + a gui typo

### PR DESCRIPTION
**Pull request by amarcolini**
# Pull Requests

Please note that we accept pull requests from anyone, but that does not mean it will be merged.

## What kind of change does this PR introduce?
* Fix
  - Video sources were improperly serialized/deserialized due to the logger not being marked as transient
  - In the settings menu, changed "Inteface" to "Interface"
* Codestyle
  - Just some code formatting
## Did this PR introduce a breaking change?
_A breaking change includes anything that breaks backwards compatibility either at compile or run time._
* No

### CODE REVIEW

File: EOCV-Sim/src/main/java/com/github/serivesmejia/eocvsim/input/InputSourceManager.java [hunk 1]
```diff
@@ -79,13 +80,13 @@ public void init() {
         inputSourceLoader.loadInputSourcesFromFile();
 
         for (Map.Entry<String, InputSource> entry : inputSourceLoader.loadedInputSources.entrySet()) {
+            logger.info("loaded input source " + entry.getKey());
```
Comment (serivesmejia) [inline EOCV-Sim/src/main/java/com/github/serivesmejia/eocvsim/input/InputSourceManager.java:83]: Really really small nitpick but maybe capitalize "loaded"?

**Review by serivesmejia — APPROVED** [review]
LGTM

**Comment by amarcolini** [comment]
Glad I could help!

**Comment by serivesmejia** [comment]
Thank you so much;)

**Pull request merged**